ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/system-control-plane.j2

```diff
@@ -0,0 +1,44 @@
+{% if system.control_plane is arista.avd.defined %}
+
+## System Control-Plane
+
+{%    if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+#### TCP MSS Ceiling
+
+| Protocol | Segment Size |
+| -------- | -------------|
+{%      if system.control_plane.tcp_mss.ipv4 is arista.avd.defined %}| ipv4 | {{ system.control_plane.tcp_mss.ipv4 }} |
+{%      endif %}
+{%      if system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}| ipv6 | {{ system.control_plane.tcp_mss.ipv6 }} |
+{%      endif %}
+{%    endif %}
+
+{%    if system.control_plane.ipv4_access_group is arista.avd.defined or system.control_plane.ipv6_access_group is arista.avd.defined %}
+#### Control-Plane Access-Groups
+
+| Protocol | VRF | Access-list |
+| -------- | --- | ------------|
+{#      IPv4 Access-groups #}
+{%      if system.control_plane.ipv4_access_group is arista.avd.defined %}
```

Comment:
This test is included in `arista.avd.natural_sort` so can be removed.

Reply:
done